backend/src/test/java/com/pickpick/acceptance/channel/ChannelSubscriptionAcceptanceTest.java

```diff
@@ -24,207 +28,223 @@
         상태코드_400_확인(response);
-        assertThat(response.jsonPath().getObject("", ErrorResponse.class).getCode()).isEqualTo(
-                "SUBSCRIPTION_NOT_EXIST");
+        에러코드_확인(response, "SUBSCRIPTION_NOT_EXIST");
     }
 
     @Test
     void 구독_채널_순서_변경_시_해당_멤버의_모든_구독_채널이_요청에_포함되지_않을_경우_예외_발생() {
         // given
+        채널_생성(MEMBER_SLACK_ID, NOTICE.create());
+        long noticeId = 1L;
+        구독_요청(token, noticeId);
+
+        채널_생성(MEMBER_SLACK_ID, FREE_CHAT.create());
+        long freeChatId = 2L;
+        구독_요청(token, freeChatId);
+
         List<ChannelOrderRequest> request = List.of(
-                new ChannelOrderRequest(channelIdToSubscribe1, 1)
+                new ChannelOrderRequest(freeChatId, 1)
         );
 
         // when
-        ExtractableResponse<Response> response = 구독_채널_순서_변경_요청(request);
+        ExtractableResponse<Response> response = 구독_채널_순서_변경_요청(token, request);
 
         // then
         상태코드_400_확인(response);
-        assertThat(response.jsonPath().getObject("", ErrorResponse.class).getCode()).isEqualTo(
-                "SUBSCRIPTION_NOT_EXIST");
+        에러코드_확인(response, "SUBSCRIPTION_NOT_EXIST");
     }
 
     @Test
     void 구독_중인_채널_다시_구독_요청() {
-        // given & when
-        ExtractableResponse<Response> response = 구독_요청(channelIdToSubscribe1);
+        // given
+        채널_생성(MEMBER_SLACK_ID, FREE_CHAT.create());
+        long freeChatId = 1L;
+        구독_요청(token, freeChatId);
+
+        // when
+        ExtractableResponse<Response> response = 구독_요청(token, freeChatId);
 
         // then
         상태코드_400_확인(response);
-        assertThat(response.jsonPath().getObject("", ErrorResponse.class).getCode()).isEqualTo(
-                "SUBSCRIPTION_DUPLICATE");
+        에러코드_확인(response, "SUBSCRIPTION_DUPLICATE");
     }
 
     @Test
     void 구독하지_않은_채널_구독_취소() {
         // given
-        구독_취소_요청(channelIdToSubscribe1);
+        채널_생성(MEMBER_SLACK_ID, FREE_CHAT.create());
+        long freeChatId = 1L;
 
         // when
-        ExtractableResponse<Response> response = 구독_취소_요청(channelIdToSubscribe1);
+        ExtractableResponse<Response> response = 구독_취소_요청(token, freeChatId);
 
         // then
         상태코드_400_확인(response);
-        assertThat(response.jsonPath().getObject("", ErrorResponse.class).getCode()).isEqualTo(
-                "SUBSCRIPTION_NOT_EXIST");
+        에러코드_확인(response, "SUBSCRIPTION_NOT_EXIST");
     }
 
-
-    private ExtractableResponse<Response> 유저_구독_채널_목록_조회_요청() {
-        return getWithToken(CHANNEL_SUBSCRIPTION_API_URL, token);
+    private List<Long> 구독한_채널_id_목록() {
+        ExtractableResponse<Response> channelsResponse = 유저_전체_채널_목록_조회_요청(token);
+        return channelsResponse.jsonPath()
+                .getList("channels.", ChannelResponse.class)
+                .stream()
+                .filter(ChannelResponse::isSubscribed)
+                .map(ChannelResponse::getId)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
    private List<Long> 구독한_채널_id_목록(final String token) {
        ExtractableResponse<Response> channelsResponse = 유저_전체_채널_목록_조회_요청(token);
        return channelsResponse.jsonPath()
                .getList("channels.", ChannelResponse.class)
                .stream()
                .filter(ChannelResponse::isSubscribed)
                .map(ChannelResponse::getId)
                .collect(Collectors.toList());
```

token을 파라미터로 받아와서 조회하는건 어떨까요? 물론 이 메서드를 사용하는 코드에서는 `구독한_채널_id_목록(token)` 보다는 `구독한_채널_id_목록()`이 깔끔하고, 현재 이 클래스에서 다른 토큰을 사용할 일이 없긴 하지만, 뭔가 토큰이 하나라고 가정하고 짠 테스트가 된 것 같아서 의견 내봅니다!